styles/Header.module.css

```diff
@@ -0,0 +1,31 @@
+.header {
```

Comment:
My only suggestion is that we move the css modules to same directory as their components. Thats the convention we and most OS project use, so lets go with that! Of course, this can be part of a different PR.

Reply:
I'll open a different PR for it next week! 

Thank for your suggestions Nikos, they were really helpful.